index.js

```diff
@@ -139,6 +140,12 @@ const generateData = function(version) {
 		'map': parsers.parseNames(version),
 		'type': 'Names'
 	}));
+	console.log('Parsing Unicode v%s Aliases…', version);
+	extend(dirMap, utils.writeFiles({
+		'version': version,
+		'map': parsers.parseAliases(version),
+		'type': 'Aliases',
```

Comment:
Should this go under the `Names` directory? Feels like it belongs there.

Reply:
Pushed an update. Here is the directory structure as reflected in the changed README:

```diff
diff --git a/README.md b/README.md
index 373e3dbadd85..c657677a7efe 100644
--- a/README.md
+++ b/README.md
@@ -57,6 +57,20 @@ const openingBrackets = require('@unicode/unicode-14.0.0/Bidi_Paired_Bracket_Typ
 Other than categories, data on Unicode properties, blocks, scripts, and script extensions is available too (for recent versions of the Unicode standard). Here’s the full list of the available data for v14.0.0:
 
 ```js
+// `Names`:
+
+require('@unicode/unicode-14.0.0/Names/index.js'); // array of canonical names
+
+require('@unicode/unicode-14.0.0/Names/Abbreviation/index.js'); // lookup map from codepoint to aliases
+
+require('@unicode/unicode-14.0.0/Names/Alternate/index.js'); // lookup map from codepoint to aliases
+
+require('@unicode/unicode-14.0.0/Names/Control/index.js'); // lookup map from codepoint to aliases
+
+require('@unicode/unicode-14.0.0/Names/Correction/index.js'); // lookup map from codepoint to aliases
+
+require('@unicode/unicode-14.0.0/Names/Figment/index.js'); // lookup map from codepoint to aliases
+
 // `Binary_Property`:
 
 require('@unicode/unicode-14.0.0/Binary_Property/ASCII/code-points.js');
```

Reply:
If you want something different let me know. I'm working with what's simple and hopefully least invasive. It's honestly a bit hard to know exactly how this is impacting the "output" library because it outputs compressed data so it's effectively diffing a huge base64 string that changes profoundly with each `npm test`. if there's a better way to validate the change please let me know.